cpp/include/cudf/functions.h

```diff
@@ -949,3 +949,26 @@ gdf_error gdf_order_by(gdf_column** input_columns,
                        size_t       num_inputs,
                        gdf_column*  output_indices,
                        int          flag_nulls_are_smallest);
+
+/* --------------------------------------------------------------------------*
+ * @brief Finds the indices of the bins in which each value of the column
+ * belongs. For `x` in `col`, if `right == false` this function finds
```

Comment:
Based on the implementation, this function won't handle null values. If this is desired behavior, make sure to indicate that this is the case.